docs/src/api/class-clock.md

```diff
@@ -107,6 +114,12 @@ await page.Clock.RunForAsync("30:00");
 
 Time may be the number of milliseconds to advance the clock by or a human-readable string. Valid string formats are "08" for eight seconds, "01:00" for one minute and "02:34:10" for two hours, 34 minutes and ten seconds.
 
+### param: Clock.runFor.ticks
+* langs: python
```

Comment:
What about C#? Should it be long there? I would think that it could be int across the langs

Reply:
Same here, no need to override